### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by DLu**
Doing some linting before other changes.

### CODE REVIEW

File: src/rqt_graph/ros_graph.py [hunk 1]
```diff
@@ -398,7 +398,7 @@ def _load_dot(self, file_name=None):
         self._widget.group_image_check_box.setEnabled(False)
         self._widget.hide_dynamic_reconfigure_check_box.setEnabled(False)
 
-        self._update_graph_view(dotcode)
+        self._update_graph_view(dotcode.decode('utf-8'))
```
Comment (clalancette) [inline src/rqt_graph/ros_graph.py:401]: This change has already been applied to the `rolling` branhc.

File: src/rqt_graph/ros_graph.py [hunk 2]
```diff
@@ -413,7 +413,7 @@ def _save_dot(self):
         if not handle.open(QIODevice.WriteOnly | QIODevice.Text):
             return
 
-        handle.write(self._current_dotcode)
+        handle.write(self._current_dotcode.encode())
```
Comment (clalancette) [inline src/rqt_graph/ros_graph.py:416]: As has this one.

**Review by clalancette — CHANGES_REQUESTED** [review]
Two things to fix, then this will be ready for CI.

File: src/rqt_graph/ros_graph.py (hunk 1 above)
Comment (DLu) [reply src/rqt_graph/ros_graph.py:401]: Might be worth deleting the ros2 branch.

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/17112/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/11660/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/17529/)

**Pull request merged**